addons/website_blog/models/website_blog.py

```diff
@@ -21,6 +21,13 @@ class Blog(models.Model):
     subtitle = fields.Char('Blog Subtitle', translate=True)
     active = fields.Boolean('Active', default=True)
     content = fields.Html('Content', translate=html_translate, sanitize=False)
+    blog_post_ids = fields.One2many('blog.post', 'blog_id', 'Blog Posts')
+    total_blog_post_number = fields.Integer("Posts", compute='_compute_total_blog_post_number', store=True)
```

Comment:
Name it `blog_post_count` to follow our convention.
Not sure I would make it a stored fields too, any need?